ocaml/xapi/export.ml

```diff
@@ -467,6 +466,33 @@ let make_pvs_sites table __context self =
   ; snapshot= API.rpc_of_pVS_site_t site
   }
 
+let make_vtpm table __context self =
+  debug "exporting vtpm %s" (Ref.string_of self) ;
+  let lookup' ref = lookup table (Ref.string_of ref) in
+  let vtpm = Db.VTPM.get_record ~__context ~self in
+  let secret =
+    Db.VTPM.get_contents ~__context ~self |> fun self ->
+    Db.Secret.get_value ~__context ~self
+  in
+  (* we are using a hand-crafted record that we serialise. The default
+     API type does not include the value of the VTPM *)
+  let vtpm' =
+    {
+      vTPM'_uuid= vtpm.vTPM_uuid
+    ; vTPM'_VM= lookup' vtpm.vTPM_VM
+    ; vTPM'_backend= vtpm.vTPM_backend
+    ; vTPM'_persistence_backend= vtpm.vTPM_persistence_backend
```

Comment:
I don't think the persistence backend has to be exported. We'll store the same data just in a different place, so it should be possible to export using XAPI backend and import using ETCD backend